lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -73,7 +73,15 @@ export class ResponseHandler {
         // Check for error
         if (serverResponseHash.error || serverResponseHash.error_description || serverResponseHash.suberror) {
             if (InteractionRequiredAuthError.isInteractionRequiredError(serverResponseHash.error, serverResponseHash.error_description, serverResponseHash.suberror)) {
-                throw new InteractionRequiredAuthError(serverResponseHash.error || Constants.EMPTY_STRING, serverResponseHash.error_description, serverResponseHash.suberror);
+                throw new InteractionRequiredAuthError(
+                    serverResponseHash.error || Constants.EMPTY_STRING,
+                    serverResponseHash.error_description,
+                    serverResponseHash.suberror,
+                    serverResponseHash.timestamp || Constants.NOT_APPLICABLE,
```

Comment:
just curios, why do we have `Constants.EMPTY_STRING` as fallback for `error` but `Constants.NOT_APPLICABLE` for others?

Reply:
Agreed, empty string is the default.

Reply:
was this actually resolved?

Reply:
This is resolved, I just hadn't pushed my changes yet.